<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="inputBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>631</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>输入</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>611</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="srcDirInput">
        <property name="spacing">
         <number>7</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <widget class="QLineEdit" name="srcDir">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="srcDirBtn">
          <property name="text">
           <string>选择文件夹..</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="evaBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>631</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="title">
     <string>结果</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>191</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>合格</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QListWidget" name="quaList">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>8</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="quaNum">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="quaNum_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>310</y>
       <width>14</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="quaOpenBtn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>330</y>
       <width>41</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>查看</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quaDirBtn">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>330</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>所在文件夹</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>191</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>不合格</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QListWidget" name="unquaList">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>8</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="unquaNum">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="unquaOpenBtn">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>330</y>
       <width>41</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>查看</string>
     </property>
    </widget>
    <widget class="QPushButton" name="unquaDirBtn">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>330</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>所在文件夹</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>191</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0" colspan="3">
       <widget class="QListWidget" name="failList">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>8</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string>操作失败</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="failNum">
        <property name="font">
         <font>
          <family>微软雅黑</family>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="failOpenBtn">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>330</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>所在文件夹</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="freshLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>570</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>正在刷新结果...</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>521</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QSlider" name="thresSlider">
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="thresSpin">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="value">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="evaBtn">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>开始筛选</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
    <property name="text">
     <string>阈值（%）</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>709</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
